bonding-rose.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#bonding-rose
+=end
+
+custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager])
+
+class BondingRose
+  include DRC
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    @settings = get_settings
+
+    unless DRStats.cleric?
+      echo '***MUST BE A CLERIC***'
+      exit
+    end
+
+    make_rose
+  end
+
+  def make_rose
+    temp_settings = @settings
+    temp_settings.hometown = 'Crossing'
+    ensure_copper_on_hand(1000, temp_settings)
+    EquipmentManager.new.empty_hands
+    unless exists?('mortar')
+      order_item(8860, 1)
+      bput('stow mortar', 'You put')
+    end
+    unless exists?('pestle')
+      order_item(8860, 3)
+      bput('stow pestle', 'You put')
+    end
+    unless exists?('mixing stick')
+      order_item(8860, 5)
+      bput('stow mix stick', 'You put')
+    end
+    unless exists?('small jar')
+      buy_item(19076, 'small jar')
+      bput('get oil in my small jar', 'You get')
+      bput('drop my oil', 'You drop')
+      bput('stow my small jar', 'You put')
+    end
+    buy_item(19076, 'oil')
```

Comment:
Is this oil different than the oil that was in the small jar (line 43)?

Reply:
I will have to double check the volumes to be sure they are the same when I'm in game next. I did it this way because on later runs when you already have a jar you don't get the free oil in the jar.

Reply:
Sounds good